llvm/test/CodeGen/AArch64/aarch64-scalarize-vec-load-ext.ll

```diff
@@ -0,0 +1,29 @@
+; RUN: llc -mtriple=aarch64-unknown-linux-gnu < %s | FileCheck %s
+
```

Comment:
Can you merge this test with whatever test was added in the original combine?

Reply:
And bonus points for using update_llc_test_checks.